android/src/main/java/com/reactnativeteadssdkmodule/RNInReadAdPlacement.kt

```diff
@@ -0,0 +1,156 @@
+package com.reactnativeteadssdkmodule
+
+import android.os.Handler
+import android.os.Looper
+import android.util.Log
+import com.facebook.react.bridge.*
+import tv.teads.sdk.*
+import tv.teads.sdk.renderer.InReadAdView
+import java.util.*
+
+
+class RNInReadAdPlacement(reactContext: ReactApplicationContext) :
```

Comment:
Is this class just a test sample?
I see the callbacks of InReadAdListener are just being logged. 
I am just wondering if this is an issue for whoever using RNInReadAdPlacement that might want to subscribe to such events

Reply:
If this is test sample would be nice to have clear path regarding what is a test path in the project
such as
`package com.reactnativeteadssdkmodule.internaltestingapp.RNInReadAdPlacement`